assets/styles/informed.scss

```diff
@@ -1954,19 +1954,19 @@ margin-bottom: 10px
 .error {
   margin-right: 15px; }
 .error .error-message {
-  color: #d4351c!important;
+  color: $govuk-error-colour !important;
   font-weight: bold; }
 .error .form-control {
-  border: 4px solid #d4351c!important; }
+  border: 4px solid $govuk-error-colour !important; }
 .error .form-hint {
   margin-bottom: 0; }
 .error-summary {
-  border-left: 4px solid #d4351c!important;;
+  border-left: 4px solid $govuk-error-colour !important;;
```

Comment:
not your code but `;;`? might as well clean this up while you're here. also... `!important`? 🤮

Reply:
Fair point